processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/GtfsFieldDescriptor.java

```diff
@@ -85,6 +87,10 @@
 
     public abstract Builder setDefaultValue(String value);
 
+    public abstract Builder setEndRange(Optional<EndRangeDescriptor> value);
```

Comment:
Is this to avoid throwing NPE?

Reply:
No, this is a convention of AutoValue. This way you can clean up the field: `setEndRange(Optional.empty())`.